ReactCommon/cxxreact/Value.h

```diff
@@ -13,6 +13,7 @@
 #include <JavaScriptCore/JSValueRef.h>
 
 #include <folly/dynamic.h>
+#include <jni/LocalString.h>
```

Comment:
Actually, looks like @cjhopman sent out an internal diff to do what I just said. Should land soon.